templates/repo_auth.conf.erb

```diff
@@ -11,6 +13,13 @@ max_num_certs_in_chain: 100
 # maintain backwards compatibility.
 verify_ssl: false
 
+# If set, this disables specific repo auth plugins. More than one plugin can be
+# specified in the form of "plugin1,plugin2,plugin3".
+# disabled_authenticators = oid_validation
+<% if @disabled_authenticators and ! @disabled_authenticators.empty? -%>
```

Comment:
Since ERB is just ruby, I think we want to avoid 'and' keyword usage in favor of '&&'.